domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/StopTimeArrivalTimeAfterDepartureTimeNotice.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity;
+
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.NoticeExporter;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.ErrorNotice;
+
+import java.io.IOException;
+
+public class StopTimeArrivalTimeAfterDepartureTimeNotice extends ErrorNotice {
+    public StopTimeArrivalTimeAfterDepartureTimeNotice(final String filename,
+                                                       final String arrivalTimeAsString,
+                                                       final String departureTimeAsString,
+                                                       final String compositeKeyFirstPart,
+                                                       final String compositeKeySecondPart,
+                                                       final String compositeKeyFirstValue,
+                                                       final String compositeKeySecondValue) {
+        super("stop_times.txt",
+                E_043,
+                "Fields `arrival_time` and `departure_time` out of order",
+                "`departure_time`: `" + departureTimeAsString + "` precedes `arrival_time`: `" +
+                        arrivalTimeAsString + "` in file `" + filename +
+                        "` for entity with composite id:" +
+                        "`" + compositeKeyFirstPart + "`: `" + compositeKeyFirstValue + "` -- " +
```

Comment:
Given that Notice class is very specific, shouldn't `*keyPart` parameters removed and string be directly built with `trip_id` and `stop_sequence` 

and `*keyValue` parameters be renamed 
`compositeKeyFirstValue` --> `tripId`
`compositeKeySecondValue` --> `stopSequence`
?